src/deploy/NVA_build/Tests.Dockerfile

```
@@ -13,7 +13,8 @@ ENV TEST_CONTAINER true
 ##############################################################
 
 RUN dnf group install -y -q "Development Tools" && \
-    dnf install -y -q --nogpgcheck vim \
+    dnf install -y -q --nogpgcheck vim && \
+    dnf install -y -q tox \
```

Why do we need another dnf enty? why not just add tox?
```suggestion
    dnf install -y -q --nogpgcheck vim  \
    tox \
```